PopPUNK/__main__.py

```diff
@@ -142,12 +143,26 @@ def get_options():
 
     # qc options
     qcGroup = parser.add_argument_group('Quality control options')
-    qcGroup.add_argument('--max-a-dist', default = 0.5, type=float, help='Maximum accessory distance to permit '
-                                                                         '[default = 0.5]')
-    qcGroup.add_argument('--ignore-length', help='Ignore outliers in terms of assembly length '
-                                                 '[default = False]', default=False, action='store_true')
-    qcGroup.add_argument('--estimated-length', default=2000000, type = int, help='Provide an integer estimated genome length when using "--ignore-length" [default = 2000000]')
-
+    qcGroup.add_argument('--qc-filter', help='Behaviour following sequence QC step: "stop" [default], "prune"'
+                                                '(analyse data passing QC), or "continue" (analyse all data)',
+                                                default='stop', type = str)
+    qcGroup.add_argument('--retain-failures', help='Retain sketches of genomes that do not pass QC filters'
```

Comment:
Redundant with above? Wouldn't choosing `continue` do this?

Reply:
`--retain-failures` siphons off the failed sketches into a separate h5 - might be useful if we want to restore sequences but don't have the original data (e.g. online submission).

Reply:
Ah I see! Can we change the help slightly
```suggestion
    qcGroup.add_argument('--retain-failures', help='Retain sketches of genomes that do not pass QC filters in a separate file'
```

Reply:
Good idea, done in 970a173.